docs/quick-install.md

```diff
@@ -1,175 +1,184 @@
-signalfx-agent status endpoints - show discovered endpoints
-signalfx-agent status monitors - show active monitors
-signalfx-agent status all - show everything
-```
+***
+ 
+#### Update your realm
+
+By default, the Smart Agent will send data to the *us0* realm. As a result, you may need to update the realm used in the installation process. To locate your realm: 
 
-#### Troubleshoot any discrepancies in the Installation 
+1. In SignalFx, in the top, right corner, click your profile icon.
+2. Click **My Profile**.
+3. Next to **Organizations**, review the listed realm.
 
-##### Realm 
+***
 
-By default, the Smart Agent will send data to the us0 realm. If you are not in this realm, you will need to explicitly set the signalFxRealm option in your config like this:
+#### Set the endpoints
 
+To explicitly set the ingest, API server, and trace endpoint URLs, review the following examples:  
 
 ```sh
-signalFxRealm: YOUR_SIGNALFX_REALM
+ingestUrl: "https://ingest.YOUR_SIGNALFX_REALM.signalfx.com"
+apiUrl: "https://api.YOUR_SIGNALFX_REALM.signalfx.com"
+traceEndpointUrl: "https://ingest.YOUR_SIGNALFX_REALM.signalfx.com/v1/trace"
 ```
 
+***
 
-To determine if you are in a different realm and need to explicitly set the endpoints, check your profile page in the SignalFx web application.
-
-_Configure your endpoints_
-
-If you want to explicitly set the ingest, API server, and trace endpoint URLs, you can set them individually like so:
+#### Review your error logs
 
+For Linux, use the following command to view error logs via Journal:
 
 ```sh
-ingestUrl: "https://ingest.YOUR_SIGNALFX_REALM.signalfx.com"
-apiUrl: "https://api.YOUR_SIGNALFX_REALM.signalfx.com"
-traceEndpointUrl: "https://ingest.YOUR_SIGNALFX_REALM.signalfx.com/v1/trace"
+journalctl -u signalfx-agent | tail -100
 ```
 
-This will default to the endpoints for the realm configured in signalFxRealm if not set.
+For Windows, simply review the event logs.
+
+***
 
-To troubleshoot your installation further, check the FAQ about troubleshooting [here](./faq.md).
+For additional installation troubleshooting information, including how to review logs, see [Frequently Asked Questions](./faq.md).
 
+***
 
-### Step 3. Login to SignalFx and discover your data displays. 
+### Review additional documentation
 
-Installation is complete.
+After a successful installation, you can learn more about:
 
-To continue your exploration of SignalFx Smart Agent capabilities, see [Advanced Installation Options](./advanced-install-options.md).
+* The capabilities of the SignalFx Smart Agent. See [Advanced Installation Options](./advanced-install-options.md).
 
-To learn more about how your data is presented in SignalFx, see the [15-Minute SignalFx Quick Start](https://docs.signalfx.com/en/latest/getting-started/quick-start.html).
+* The SignalFx UI, including how data is displayed. See [15-Minute SignalFx Quick Start](https://docs.signalfx.com/en/latest/getting-started/quick-start.html).
```

Comment:
I'm not a fan of bullets that start with sentence fragments. So I don't really see this as an improvement on the old text, except that adding bullets would be nice.

For the quick start, you should take them right to the target (https://docs.signalfx.com/en/latest/getting-started/quick-start.html#step-3-view-infrastructure-status). Because it dumps them at  Step 3, so it might be good to say something like:

To learn more about how your data is presented in SignalFx, start with [this section](...html#target) in the **15-Minute SignalFx Quick Start**.

Reply:
It's a very short section, so there wasn't much to improve.; at least the content flows better now, in my opinion. I'll update the link, and turn the sentence fragments into a complete sentence.